lib/avo/fields/belongs_to_field.rb

```diff
@@ -229,22 +238,22 @@ def database_id
         id
       end
 
-      def target_resource
+      def target_resource(record: @record, polymorphic_model_class: value&.class)
```

Comment:
Also, I'd like to maintain `target_resource` as a method without params so we can quickly call `field.target_resource` when we need to.